projects/common/src/lib/elements/manage/manage.component.html

```diff
@@ -185,8 +192,30 @@ <h2>Enroll a New Device</h2>
                   placeholder="The Name of your Device"
                   required
                 />
+                
               </mat-form-field>
+              
             </div>
+            <div *ngIf="AddDeviceFormGroup.get('deviceName').invalid && (AddDeviceFormGroup.get('deviceName').dirty || AddDeviceFormGroup.get('deviceName').touched)"

```

Comment:
I'm not a big fan of this much logic inside the HTML, and feel as though it should be covered by props in the .ts file:

See below for code on simplifying ErrorText, then use:
`<div *ngIf="DeviceNameErrorText">...</div>`